packages/sdk-core/.eslintrc.json

```diff
@@ -1,10 +1,57 @@
 {
```

Comment:
This configuration was not proper for TypeScript previously. I copied the one from sdk-redux here (which was initially created by scaffolding a TS library).